config/webpack-config.js

```diff
@@ -12,8 +12,10 @@ const scriptsManifest = require( '../gulp/utils/scripts-manifest' );
 const webpack = require( 'webpack' );
 const UglifyWebpackPlugin = require( 'uglifyjs-webpack-plugin' );
 
+
 // Constants.
 const JS_ROUTES_PATH = '/js/routes';
+const OAH_COMMON_BUNDLE_ROUTE = '/js/routes/owning-a-home/';
```

Comment:
So is this a temp thing? I'm unclear why we couldn't include a `index.js` to deliver bundles to the OAH pages?  https://github.com/cfpb/cfgov-refresh/blame/master/cfgov/jinja2/v1/_layouts/base.html#L262